tsconfig.json

```diff
@@ -13,7 +13,6 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "jsx": "preserve",
-    "noImplicitAny": false,
```

Comment:
removing this opens a large can of worms

![](https://media.giphy.com/media/5z2cV4tWz04P4BQj9Z/giphy-downsized.gif)